--- doxygen-1.5.1/addon/doxywizard/Makefile.doxywizard~	2006-12-11 17:04:59.000000000 +0200
+++ doxygen-1.5.1/addon/doxywizard/Makefile.doxywizard	2006-12-11 17:05:22.000000000 +0200
@@ -24,7 +24,7 @@
 INCPATH	=	-I../../src -I$(QTDIR)/include
 LINK	=	c++
 LFLAGS	=	-Wl,-search_paths_first
-LIBS	=	-framework CoreFoundation -framework ApplicationServices -L$(QTDIR)/lib -lqt
+LIBS	=	-framework CoreFoundation -framework ApplicationServices -L$(QTDIR)/lib64 -lqt
 MOC	=	moc
 
 TAR	=	tar -cf
--- doxygen-1.4.2/tmake/doc/m-linux-gcc.html	2003-12-14 23:00:58.000000000 +0100
+++ doxygen-1.4.2.new/tmake/doc/m-linux-gcc.html	2005-03-28 19:29:53.126157744 +0200
@@ -19,7 +19,7 @@
 INCPATH	=	-I$(QTDIR)/include
 LINK	=	g++
 LFLAGS	=	
-LIBS	=	-L$(QTDIR)/lib -lqt -L/usr/X11R6/lib -lX11
+LIBS	=	-L$(QTDIR)/lib64 -lqt -L/usr/X11R6/lib64 -lX11
 MOC	=	moc
 
 ####### Files
--- doxygen-1.4.6/configure.orig	2006-01-12 20:39:52.000000000 +0100
+++ doxygen-1.4.6/configure	2006-01-12 20:43:06.000000000 +0100
@@ -243,7 +243,7 @@
 
 #- check for qt --------------------------------------------------------------
  
-LIBDIR=lib
+LIBDIR=lib64
 if test "$f_wizard" = YES; then
   if test -z "$QTDIR"; then
     echo "  QTDIR environment variable not set!"
--- doxygen-1.4.2/tmake/lib/linux-g++/tmake.conf.orig	2005-03-28 21:38:45.428533235 +0200
+++ doxygen-1.4.2/tmake/lib/linux-g++/tmake.conf	2005-03-28 21:39:29.538941670 +0200
@@ -28,11 +28,11 @@
 TMAKE_INCDIR		=
 TMAKE_LIBDIR		=
 TMAKE_INCDIR_X11	= /usr/X11R6/include
-TMAKE_LIBDIR_X11	= /usr/X11R6/lib
+TMAKE_LIBDIR_X11	= /usr/X11R6/lib64
 TMAKE_INCDIR_QT		= $(QTDIR)/include/qt
-TMAKE_LIBDIR_QT		= $(QTDIR)/lib
+TMAKE_LIBDIR_QT		= $(QTDIR)/lib64
 TMAKE_INCDIR_OPENGL	= /usr/X11R6/include
-TMAKE_LIBDIR_OPENGL	= /usr/X11R6/lib
+TMAKE_LIBDIR_OPENGL	= /usr/X11R6/lib64
 
 TMAKE_LINK		= g++
 TMAKE_LINK_SHLIB	= g++
