diff -up doxygen-1.6.0/addon/doxywizard/Makefile.in.config doxygen-1.6.0/addon/doxywizard/Makefile.in
--- doxygen-1.6.0/addon/doxywizard/Makefile.in.config	2009-07-13 22:09:03.000000000 +0200
+++ doxygen-1.6.0/addon/doxywizard/Makefile.in	2009-08-21 16:42:13.000000000 +0200
@@ -10,8 +10,6 @@
 # See the GNU General Public License for more details.
 # 
 
-QMAKE=qmake $(MKSPECS)
-
 all: Makefile.doxywizard  
 	$(MAKE) -f Makefile.doxywizard 
 
--- doxygen-1.8.4/configure.orig	2013-05-19 12:05:39.000000000 +0200
+++ doxygen-1.8.4/configure	2013-06-08 09:12:44.470080015 +0200
@@ -296,9 +296,10 @@
   if test -z "$QTDIR"; then
     echo "  QTDIR environment variable not set!"
     printf "  Checking for Qt..."
-    for d in /usr/{lib,share,qt}/{qt-4,qt4,qt,qt*,4} /usr; do
+    for d in /usr/{lib64,lib,share,qt}/{qt-4,qt4,qt,qt*,4}; do
       if test -x "$d/bin/qmake"; then
         QTDIR=$d
+        QMAKE=$d/bin/qmake
       fi
     done
   else
@@ -606,6 +607,8 @@
 DOXYDOCS  = ..
 DOCDIR    = $f_docdir
 QTDIR     = $QTDIR
+QMAKE     = $QMAKE
+MAN1DIR    = share/man/man1
 EOF
 
 if test "$f_dot" != NO; then
--- doxygen-1.8.3/Makefile.in.orig	2013-01-02 21:33:10.021332601 +0100
+++ doxygen-1.8.3/Makefile.in	2013-01-03 16:32:34.159907679 +0100
@@ -53,8 +53,6 @@
 
 DATE=$(shell date "+%B %Y")
 
-MAN1DIR = man/man1
-
 install: doxywizard_install doxysearch_install
 	$(INSTTOOL) -d $(DESTDIR)/$(INSTALL)/bin
 	$(INSTTOOL) -m 755 bin/doxygen        $(DESTDIR)/$(INSTALL)/bin
