diff -ur doxygen-1.4.2/addon/doxywizard/Makefile.doxywizard doxygen-1.4.2.new/addon/doxywizard/Makefile.doxywizard
--- doxygen-1.4.2/addon/doxywizard/Makefile.doxywizard	2005-03-28 10:38:39.000000000 +0200
+++ doxygen-1.4.2.new/addon/doxywizard/Makefile.doxywizard	2005-03-28 19:28:46.000362416 +0200
@@ -24,7 +24,7 @@
 INCPATH	=	-I../../src -I$(QTDIR)/include
 LINK	=	g++
 LFLAGS	=	-g
-LIBS	=	 -L$(QTDIR)/lib -lqt-mt -L/usr/X11R6/lib -lXext -lX11 -lm
+LIBS	=	 -L$(QTDIR)/lib64 -lqt-mt -L/usr/X11R6/lib64 -lXext -lX11 -lm
 MOC	=	/usr/qt/3/bin/moc
 
 TAR	=	tar -cf
diff -ur doxygen-1.4.2/configure doxygen-1.4.2.new/configure
--- doxygen-1.4.2/configure	2005-01-03 20:23:19.000000000 +0100
+++ doxygen-1.4.2.new/configure	2005-03-28 19:29:30.973525456 +0200
@@ -247,8 +247,8 @@
     echo "check the Qt installation instructions!"
     exit 2
   else
-    if test ! -d "$QTDIR/lib"; then
-      echo "QTDIR is set to $QTDIR, but library directory $QTDIR/lib does not exist!"
+    if test ! -d "$QTDIR/lib64"; then
+      echo "QTDIR is set to $QTDIR, but library directory $QTDIR/lib64 does not exist!"
       exit 2
     fi
     if test ! -d "$QTDIR/include"; then
@@ -260,8 +260,8 @@
       exit 2
     fi
     echo " headers $QTDIR/include,"
-    echo "                     libraries $QTDIR/lib"  
-    if test -n "`ls $QTDIR/lib/* | grep qt-mt`"; then
+    echo "                     libraries $QTDIR/lib64"  
+    if test -n "`ls $QTDIR/lib64/* | grep qt-mt`"; then
       f_thread=YES
     fi
   fi
diff -ur doxygen-1.4.2/tmake/doc/m-linux-gcc.html doxygen-1.4.2.new/tmake/doc/m-linux-gcc.html
--- doxygen-1.4.2/tmake/doc/m-linux-gcc.html	2003-12-14 23:00:58.000000000 +0100
+++ doxygen-1.4.2.new/tmake/doc/m-linux-gcc.html	2005-03-28 19:29:53.126157744 +0200
@@ -19,7 +19,7 @@
 INCPATH	=	-I$(QTDIR)/include
 LINK	=	g++
 LFLAGS	=	
-LIBS	=	-L$(QTDIR)/lib -lqt -L/usr/X11R6/lib -lX11
+LIBS	=	-L$(QTDIR)/lib64 -lqt -L/usr/X11R6/lib64 -lX11
 MOC	=	moc
 
 ####### Files
diff -ur doxygen-1.4.2/tmake/lib/linux-g++/tmake.conf doxygen-1.4.2.new/tmake/lib/linux-g++/tmake.conf
--- doxygen-1.4.2/tmake/lib/linux-g++/tmake.conf	2004-06-20 18:50:17.000000000 +0200
+++ doxygen-1.4.2.new/tmake/lib/linux-g++/tmake.conf	2005-03-28 19:30:52.187179104 +0200
@@ -28,11 +28,11 @@
 TMAKE_INCDIR		=
 TMAKE_LIBDIR		=
 TMAKE_INCDIR_X11	= /usr/X11R6/include
-TMAKE_LIBDIR_X11	= /usr/X11R6/lib
+TMAKE_LIBDIR_X11	= /usr/X11R6/lib64
 TMAKE_INCDIR_QT		= $(QTDIR)/include
-TMAKE_LIBDIR_QT		= $(QTDIR)/lib
+TMAKE_LIBDIR_QT		= $(QTDIR)/lib64
 TMAKE_INCDIR_OPENGL	= /usr/X11R6/include
-TMAKE_LIBDIR_OPENGL	= /usr/X11R6/lib
+TMAKE_LIBDIR_OPENGL	= /usr/X11R6/lib64
 
 TMAKE_LINK		= g++
 TMAKE_LINK_SHLIB	= g++
