--- doxygen-1.8.6/configure.orig	2013-12-24 17:14:46.000000000 +0100
+++ doxygen-1.8.6/configure	2014-01-29 20:59:58.925766639 +0100
@@ -326,7 +326,7 @@
   if test -z "$QTDIR"; then
     echo "  QTDIR environment variable not set!"
     printf "  Checking for Qt..."
-    for d in /usr/{lib,share,qt}/{qt-4,qt4,qt,qt*,4} /usr; do
+    for d in /usr/{lib64,lib,share,qt}/{qt-4,qt4,qt,qt*,4}; do
       if test -x "$d/bin/qmake"; then
         QTDIR=$d
         break 2
@@ -704,6 +704,7 @@
 DOXYDOCS  = ..
 DOCDIR    = $f_docdir
 QTDIR     = $QTDIR
+MAN1DIR    = share/man/man1
 EOF
 
 if test "$f_dot" != NO; then
--- doxygen-1.8.3/Makefile.in.orig	2013-01-02 21:33:10.021332601 +0100
+++ doxygen-1.8.3/Makefile.in	2013-01-03 16:32:34.159907679 +0100
@@ -53,8 +53,6 @@
 
 DATE=$(shell date "+%B %Y")
 
-MAN1DIR = man/man1
-
 install: doxywizard_install doxysearch_install
 	$(INSTTOOL) -d $(DESTDIR)/$(INSTALL)/bin
 	$(INSTTOOL) -m 755 bin/doxygen        $(DESTDIR)/$(INSTALL)/bin
