diff -up doxygen-1.6.0/addon/doxywizard/Makefile.in.config doxygen-1.6.0/addon/doxywizard/Makefile.in
--- doxygen-1.6.0/addon/doxywizard/Makefile.in.config	2009-07-13 22:09:03.000000000 +0200
+++ doxygen-1.6.0/addon/doxywizard/Makefile.in	2009-08-21 16:42:13.000000000 +0200
@@ -10,8 +10,6 @@
 # See the GNU General Public License for more details.
 # 
 
-QMAKE=qmake
-
 all: Makefile.doxywizard  
 	$(MAKE) -f Makefile.doxywizard 
 
@@ -29,11 +27,11 @@ distclean: Makefile.doxywizard
 	$(RM) Makefile.doxywizard
 
 install:
-	$(INSTTOOL) -d $(INSTALL)/bin	
-	$(INSTTOOL) -m 755 ../../bin/doxywizard $(INSTALL)/bin	
-	$(INSTTOOL) -d $(INSTALL)/$(MAN1DIR)
+	$(INSTTOOL) -d $(DESTDIR)$(INSTALL)/bin	
+	$(INSTTOOL) -m 755 ../../bin/doxywizard $(DESTDIR)$(INSTALL)/bin	
+	$(INSTTOOL) -d $(DESTDIR)$(INSTALL)/$(MAN1DIR)
 	cat ../../doc/doxywizard.1 | sed -e "s/DATE/$(DATE)/g" -e "s/VERSION/$(VERSION)/g" > doxywizard.1
-	$(INSTTOOL) -m 644 doxywizard.1 $(INSTALL)/$(MAN1DIR)/doxywizard.1
+	$(INSTTOOL) -m 644 doxywizard.1 $(DESTDIR)$(INSTALL)/$(MAN1DIR)/doxywizard.1
 	rm doxywizard.1
 
 FORCE:
diff -up doxygen-1.6.0/configure.config doxygen-1.6.0/configure
--- doxygen-1.6.0/configure.config	2009-08-20 14:15:07.000000000 +0200
+++ doxygen-1.6.0/configure	2009-08-21 16:41:36.000000000 +0200
@@ -268,9 +268,10 @@ if test "$f_wizard" = YES; then
   if test -z "$QTDIR"; then
     echo "  QTDIR environment variable not set!"
     echo -n "  Checking for Qt..."
-    for d in /usr/{lib,share,qt}/{qt-4,qt4,qt,qt*,4}; do
+    for d in /usr/{lib64,lib,share,qt}/{qt-4,qt4,qt,qt*,4}; do
       if test -x "$d/bin/qmake"; then
         QTDIR=$d
+        QMAKE=$d/bin/qmake
       fi
     done
   else
@@ -485,6 +486,8 @@ INSTTOOL  = $f_insttool
 DOXYDOCS  = ..
 DOCDIR    = $f_docdir
 QTDIR     = $QTDIR
+QMAKE     = $QMAKE
+MAN1DIR    = share/man/man1
 EOF
 
 if test "$f_dot" != NO; then
diff -up doxygen-1.6.0/Makefile.in.config doxygen-1.6.0/Makefile.in
--- doxygen-1.6.0/Makefile.in.config	2009-08-14 15:58:42.000000000 +0200
+++ doxygen-1.6.0/Makefile.in	2009-08-21 16:34:02.000000000 +0200
@@ -44,8 +44,6 @@ distclean: clean
 
 DATE=$(shell date "+%B %Y")
 
-MAN1DIR = man/man1
-
 install: doxywizard_install 
 	$(INSTTOOL) -d $(DESTDIR)/$(INSTALL)/bin
 	$(INSTTOOL) -m 755 bin/doxygen    $(DESTDIR)/$(INSTALL)/bin
